src/react-components/styles/theme.js

```diff
@@ -37,90 +35,84 @@ function useDarkMode() {
     [setDarkMode]
   );
 
-  useEffect(
-    () => {
-      const darkmodeQuery = window.matchMedia("(prefers-color-scheme: dark)");
-
-      setDarkMode(darkmodeQuery.matches);
+  useEffect(() => {
+    const darkmodeQuery = window.matchMedia("(prefers-color-scheme: dark)");
+
+    setDarkMode(darkmodeQuery.matches);
+
+    // This is a workaround for old Safari.
+    // Prior to Safari 14, MediaQueryList is based on EventTarget, so you must use
+    // addListener() and removeListener() to observe media query lists.
+    // https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList/addListener
+    // We may remove this workaround when no one will use Safari 13 or before.
+    if (darkmodeQuery.addEventListener) {
+      darkmodeQuery.addEventListener("change", changeListener);
+    } else {
+      darkmodeQuery.addListener(changeListener);
+    }
 
-      // This is a workaround for old Safari.
-      // Prior to Safari 14, MediaQueryList is based on EventTarget, so you must use
-      // addListener() and removeListener() to observe media query lists.
-      // https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList/addListener
-      // We may remove this workaround when no one will use Safari 13 or before.
-      if (darkmodeQuery.addEventListener) {
-        darkmodeQuery.addEventListener("change", changeListener);
+    return () => {
+      if (darkmodeQuery.removeEventListener) {
+        darkmodeQuery.removeEventListener("change", changeListener);
       } else {
-        darkmodeQuery.addListener(changeListener);
+        darkmodeQuery.removeListener(changeListener);
       }
-
-      return () => {
-        if (darkmodeQuery.removeEventListener) {
-          darkmodeQuery.removeEventListener("change", changeListener);
-        } else {
-          darkmodeQuery.removeListener(changeListener);
-        }
-      };
-    },
-    [changeListener]
-  );
+    };
+  }, [changeListener]);
 
   return darkMode;
 }
 
 export function useTheme(themeId) {
   const darkMode = useDarkMode();
 
-  useEffect(
-    () => {
-      // Themes can come from an external source. Ensure it is an array.
-      if (!Array.isArray(themes)) return;
+  useEffect(() => {
+    // Themes can come from an external source. Ensure it is an array.
+    if (!Array.isArray(themes)) return;
 
-      let theme;
+    let theme;
 
-      if (themeId) {
-        theme = themes.find(t => t.id === themeId);
-      }
+    if (themeId) {
+      theme = themes.find(t => t.id === themeId);
+    }
 
-      if (!theme && darkMode) {
-        theme = themes.find(t => t.darkModeDefault);
-      }
+    if (!theme && darkMode) {
+      theme = themes.find(t => t.darkModeDefault);
+    }
 
-      if (!theme) {
-        theme = themes.find(t => t.default);
-      }
+    if (!theme) {
+      theme = themes.find(t => t.default);
+    }
```

Comment:
could add a return in this (!then) and remove the next one.

Reply:
This suggestion makes sense, but this code only happened to change because of code formatting. The actual change in this file was making the `APP` lookup optional (to fix storybook). Gunna keep this as is in this PR.